internal/resource/configmap_test.go

```diff
@@ -252,13 +296,66 @@ listeners.ssl.default = 5671
 ssl_options.certfile   = /etc/rabbitmq-tls/tls.crt
 ssl_options.keyfile    = /etc/rabbitmq-tls/tls.key
 listeners.ssl.default  = 5671
+
+management.ssl.certfile   = /etc/rabbitmq-tls/tls.crt
+management.ssl.keyfile    = /etc/rabbitmq-tls/tls.key
+management.ssl.port       = 15671
+
 ssl_options.cacertfile = /etc/rabbitmq-tls/ca.certificate
 ssl_options.verify     = verify_peer
+management.ssl.cacertfile = /etc/rabbitmq-tls/ca.certificate
 `)
 
 				Expect(configMapBuilder.Update(configMap)).To(Succeed())
 				Expect(configMap.Data).To(HaveKeyWithValue("rabbitmq.conf", expectedRabbitmqConf))
 			})
+
+			When("additional plugins are enabled", func() {
+				It("adds TLS config for the additional plugins", func() {
```

Comment:
We can have the description a bit clearer that this test is about testing the tls config for `rabbitmq_web_mqtt` and `rabbitmq_web_stomp`. Right now the spec description reads the same as the one from line `239`.

Reply:
Ah good spot. I agree we can be more clear here, thanks Chunyi!